apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -45,6 +49,7 @@ public static CSVParser createCsvParser(RifFile file) {
    */
   public static CSVParser createCsvParser(CSVFormat csvFormat, RifFile file) {
     String displayName = file.getDisplayName();
+    LOGGER.info(String.format("File: {0} is ready for csv parser", displayName));
```

Comment:
I was hoping to only log if we had to mod / remove backlash. This will log no matter what, I think. You'd have to read the file again, which would add time. I think we can just script something when we need to check back on this for occurances.

Reply:
Taken out